test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -45,6 +45,16 @@ struct UcxAmDesc {
     const ucp_am_recv_param_t    *_param;
 };
 
+void *ucx_am_get_data(const UcxAmDesc &desc)
```

Comment:
no need this function, better remove and use desc.data directly when needed

Reply:
UcxAmDesc is defined in ucx_wrapper.cc and invisible to io_demo.cc. That is the reason why we needed wrapper functions here. Moving UcxAmDesc to ucx_wrapper.h seemed not what Mikhail wanted. @brminich Could you chime in?

Reply:
imo, using the func looks better

Reply:
so need to make it protected static inline

Reply:
done